hubmap/query_app/utils.py

```diff
@@ -128,3 +146,29 @@ def split_at_comparator(item: str) -> List:
             item_split.insert(1, comparator)
             return item_split
     return []
+
+
+def get_response_from_query_handle(query_handle: str, set_type: str):
+    query_dict = OrderedDict()
```

Comment:
All `dict`s are ordered in Python >= 3.7; this was an implementation detail in CPython 3.6 and added to the language spec in 3.7. May as well do `query_dict = {}`. (also three other occurrences in this diff)